### ISSUE TITLE
Buffer getting exposed on frontend - StampRelatedGraph

### ISSUE THREAD

**Issue by reinamora137** [stampchain-io/BTCStampsExplorer#182]
there is a buffer import happening on the frontend to the stamp detail page.

0e403b069a6cfb9bebbf4888bbd0dbc06af211f34c4deeb9ccfe43e3f9d8cc9c:1 Access to script at 'node:buffer' from origin 'http://localhost:8002' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: brave, chrome, chrome-extension, chrome-untrusted, data, http, https, isolated-app.

**Comment by reinamora137** [comment]
when removing StampRelatedGraph from `routes/stamp/[id].tsx`  the problem is resolved

```
        {
          /* <StampRelatedGraph
          stamp={stamp}
        /> */
        }
```

This could be related to fixes pending in #169 and #158 

definitely issues with the Graph that need to be resolved

**Comment by reinamora137** [comment]
this appears to have been resolved, still some display issues with the graph
